frontend-ssr/pages/_app.tsx

```diff
@@ -12,6 +13,10 @@ export default function MyApp({ Component, pageProps }: any) {
     },
   });
 
+  if (typeof window !== 'undefined' && process.env.NODE_ENV === 'production') {
```

Comment:
Can we use the `useEffect` hook here to simulate `componentDidMount`? This way we don't need to check for `window !== 'undefined'`

Reply:
Also, I just realized that we should lazy load this and dynamically import it in the same `useEffect` hook. This way we can save 32KB - https://bundlephobia.com/result?p=logrocket@1.0.14 - from the initial bundle.

Reply:
Don't forget about the "privacy" page as well. If logrocket is using cookies we should add it.